spec/models/charge_filter_spec.rb

```diff
@@ -344,4 +344,23 @@
       end
     end
   end
+
+  describe '#to_h' do
+    subject(:charge_filter) { build(:charge_filter, values:) }
+
+    let(:billable_metric_filter) { create(:billable_metric_filter, key: 'card') }
+    let(:values) do
+      [
+        build(:charge_filter_value, value: 'mastercard', billable_metric_filter:),
+        build(:charge_filter_value, value: 'visa', billable_metric_filter:),
+      ]
+    end
+
+    it 'returns the values as a hash' do
+      expect(charge_filter.to_h).to eq({
```

Comment:
Layout/FirstMethodArgumentLineBreak: Add a line break before the first argument of a multi-line method argument list.